spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -170,7 +215,8 @@ def matcher
   def validating_presence(options = {})
     define_model :example, attr: :string do
       validates_presence_of :attr, options
-    end.new
+    end
+    .new
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.